mobile/src/store/uam/actions.ts

```diff
@@ -27,12 +28,18 @@ const deleteGroup = createAsyncThunk<
   number,
   GroupsDeleteRequestParamDto,
   AsyncThunkConfig
->(ActionType.DELETE_GROUP, async (payload, { extra }) => {
+>(ActionType.DELETE_GROUP, async (payload, { extra, getState, dispatch }) => {
+  const currentUser = getState().auth.user?.id;
   const { groupsApi } = extra;
-
+  const { id } = payload;
+  const groupsInfo = await groupsApi.getById(payload);
   await groupsApi.delete(payload);
 
-  const { id } = payload;
+  if (currentUser) {
+    if (groupsInfo.userIds.includes(currentUser)) {
```

Comment:
```suggestion
    if (groupsInfo.userIds.includes(currentUser?.id)) {
```

Reply:
Hmm...something like that ?
![image](https://user-images.githubusercontent.com/82529236/185910680-e50b22e3-31d0-4d86-97be-02a788ef3660.png)